Makefile

```diff
@@ -39,17 +39,13 @@ endif
 LD_FLAGS = $(BUILTIN_LD_FLAGS) $(EXTRA_LD_FLAGS)
 
 # BUILTIN_GC_FLAGS are the internal flags used to pass compiler.
-BUILTIN_GC_FLAGS ?= all=-trimpath="$$HOME"
+BUILTIN_GC_FLAGS ?=
```

Comment:
Perhaps this could go back to the `BUILTIN_GC_FLAGS =` form that it was in prior to #3443?  (Or even be removed entirely, but it seems like it was added intentionally in that form in 2235198650d8dfadded525cc308ae8dfa786da52 of PR #3144.)